x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -285,12 +285,17 @@ export const createMetricThresholdExecutor = (libs: InfraBackendLibs) =>
             ? WARNING_ACTIONS_ID
             : FIRED_ACTIONS_ID;
 
-        const additionalContext = hasAdditionalContext(params.groupBy, validGroupByForContext)
+        let additionalContext = hasAdditionalContext(params.groupBy, validGroupByForContext)
           ? alertResults && alertResults.length > 0
             ? alertResults[0][group].context
             : null
```

Comment:
nit: I think I would just define a const `{ tags: [] }`, and use that instead of `null` here, so you won't have to use a `let`.  If I understand the logic correctly.